src/components/admin/lib/article-validators.js

```diff
@@ -0,0 +1,24 @@
+export function checkForHttp(html) {
+  if (/http:\/\/.*\..{3,}/.test(html) || /http(?!s)/.test(html)) {
```

Comment:
I think the `.` is a problem in this regex, for the wildcards you should use `\S` (which means anything but whitespace), as otherwise you could match all kinds of crazy things, might be even better to do something like `[^\s"']` or any other characters that cannot be, or maybe find exactly what is allowed in a url such as `[a-zA-Z\.-_]` or w/e.